src/apps/VOL3D.cpp

```diff
@@ -25,15 +25,15 @@ namespace apps
 VOL3D::VOL3D(const RunParams& params)
   : KernelBase(rajaperf::Apps_VOL3D, params)
 {
-  setDefaultSize(100*100*100);  // See rzmax in ADomain struct
+  setDefaultProblemSize(100*100*100);  // See rzmax in ADomain struct
   setDefaultReps(100);
 
-  Index_type rzmax = std::cbrt(getRunSize())+1;
+  Index_type rzmax = std::cbrt(getTargetProblemSize())+1;
   m_domain = new ADomain(rzmax, /* ndims = */ 3);
 
   m_array_length = m_domain->nnalls;
 
-  setProblemSize( m_domain->n_real_zones );
+  setActualProblemSize( m_domain->lpz+1 - m_domain->fpz );
```

Comment:
I set this to the number of zones because that's how we think of problem size, but this is probably more consistent for the purposes of the perf suite.

Reply:
I can go either way on this. I believe my change is consistent with the loop size in the kernel.  Change it back?

Reply:
Let's keep it, its more consistent to take the loop size here than the zone count.